CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 1.2.0
+
+TODO: Add the missing PRs here once merged!
+
+- feat: #65 Custom timestamp parameter (#254) (2024-06-19)
```

Comment:
Just wondering if the date is necessary if this all gets merged at once. Perhaps just one date?

Reply:
I will remove the dates.